portfolio/src/main/webapp/index.html

```diff
@@ -5,11 +5,22 @@
     <meta name="viewport" content="initial-scale=1.0, user-scalable=no">
     <title>Lee's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@300&family=Muli:wght@300&display=swap" rel="stylesheet">
     <script src="script.js"></script>
+    <!-- font style -->
+    <link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@300&family=Muli:wght@300&display=swap" rel="stylesheet">
+    <!-- contact icons  -->
     <script src="https://kit.fontawesome.com/d691fbec03.js" crossorigin="anonymous"></script>
-    <!-- <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDTkaBUiXmTof1JeMXiplYaeTZ39ms4DNU&callback=initMap" -->
-  type="text/javascript"></script>
+    <!-- bootstrap -->
+    <!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css"> -->
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <!-- Latest compiled JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <!-- Google Platform for sign in   -->
+    <script src="https://apis.google.com/js/platform.js" async defer></script>
```

Comment:
I assume you're going to use these scripts at some point?

Reply:
Yup! I A stretch goal of mine is to implement Bootstrap to help with the mobile view, and the second script is already in use with Google sign in.